doc/commands/rcheckin.md

```diff
@@ -60,6 +60,8 @@ rcheckin differs from [checkin](checkin.md) in that the latter squashes the comm
                                    synchronization at some point. Use this option
                                    with care. This option is ignored if ignore-
                                    missing-files option is used.
+      -wo, --workspace-owner=VALUE1
```

Comment:
Thanks for the tip, I didn't know about Linux command-line tools! I fixed the option here 5e986260d531e5b11c246224add5a6bbdcdb33df

Reply:
@ecomachio I don't have a clue on how this feature could be used. And if we accept to include this feature it is so that it could be useful for someone else. In that purpose, I think that some documentation on how to use it (especially what is the type of value that should be passed) and an example of use would be needed...

Reply:
@pmiossec thank you for your input!

I can add a more detailed example of the use in the checkin doc. 

Just to explain to you a little more... this feature is useful when you have an automated process on a server committing to a TFS repo. 

For instance, in my case, we have a server that keeps the TFS and the GIT repo synchronized. But our TFS repo has some policies we need to satisfy before committing, one of them is the compilation of the files being committed and the tool that compiles these files copy them by getting the workspace associated with the author of the commit.

I added some examples of use in the docs here 2c741724184687c620255495d5924b5b64bd9a4d

I don't know if is that enough tough

Thank you!

Reply:
Would this also allow to commit to TFS as someone else? E.g. saying that my normal user name is `mickey.mouse`,
would this feature allow to impersonate the TFS user `donald.duck` so all changes committed to TFS will appear
as `donald.duck` instead?

Reply:
To commit as someone else, like you're saying, you can use the --author option. but if you have a checkin police like in my case you will need the workspace in name of the author as well.

The user which is running the command must have the right permissions as well or else you will get a TF14044 error.

in my case the command is like this 

> `git tfs checkin -m "comment"  -n "checkinnotes" --workspace-owner  DOMAIN\edian.comachio -author DOMAIN\edian.comachio -d`

So with this line we have one Adminstrator user having all privileges needed, who can make commits on behalf of the all user with less permissions.